jd4/api.py

```diff
@@ -57,10 +57,21 @@ def full_url(self, *parts):
     async def judge_consume(self, handler_type):
         async with self.ws_connect(self.full_url('judge/consume-conn/websocket')) as ws:
             logger.info('Connected')
-            async for msg in ws:
-                request = json.loads(msg.data)
-                await handler_type(self, request, ws).handle()
-            logger.warning('Connection lost with code %d', ws.close_code)
+            queue = Queue()
+            async def fetch():
```

Comment:
1. We can create only one task instead of two, and call `ws.close()` on exception.

```
async def worker():
    try:
        while True:
            request = await queue.get()
            await ...
    except Exception as e:
        log...
        ws.close()
```

2. We can allow some level of parallelism, i.e. start multiple workers, maybe in the future.
3. We can use the new [`receive_json`](https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientWebSocketResponse.receive_json).